sdk/attestation/azure-security-attestation/src/attestation_administration_client.cpp

```diff
@@ -534,7 +534,6 @@ AttestationAdministrationClient::RemoveIsolatedModeCertificate(
  * @details Validating the response returned by the attestation service requires a set of
  * possible signers for the attestation token.
  *
```

Comment:
I don't think we need this comment block at all in `.cpp` files. It is redundant and could create confusion. We only need such documentation in the corresponding `.hpp` file, which we already have.